content/docs/started/k8s/overview.md

```diff
@@ -13,18 +13,81 @@ existing Kubernetes cluster is from one of those, consider following those instr
 
 The Kubernetes cluster must meet the following minimum requirements:
 
+  * Your cluster must include at least one worker node with a minimum of:
+    * 4 CPU
+    * 50 GB storage
+    * 12 GB memory
+
+
   * The recommended Kubernetes version is {{% kubernetes-tested-version %}}.
     Kubeflow has been validated and tested on Kubernetes
     {{% kubernetes-tested-version %}}.
     * Your cluster must run at least Kubernetes version 
     {{% kubernetes-min-version %}}. 
     * Kubeflow **does not work** on Kubernetes 
       {{% kubernetes-incompatible-versions %}}.
+    * Older versions of Kubernetes may not be compatible with the latest Kubeflow versions.  The following matrix
+    provides information about compatibility between Kubeflow and Kubernetes versions.
 
-  * Your cluster must include at least one worker node with a minimum of:
-    * 4 CPU
-    * 50 GB storage
-    * 12 GB memory
+    
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Kubernetes Versions</th>
+        <th>Kubeflow 0.4</th>
+        <th>Kubeflow 0.5</th>
+        <th>Kubeflow 0.6</th>
+        <th>Kubeflow 0.7</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>1.11</td>
+        <td>compatible</td>
+        <td>compatible</td>
+        <td>incompatible</td>
```

Comment:
It's a little hard to pick out the incompatible entries. For now, let's make them bold:
```
<td><b>incompatible</b></td>
```

It would be nice to have a more visual grid. For example a tick (check mark) for compatible, and a cross for incompatible. That's likely to be some work. We'd likely have to explore Bootstrap and make a shortcode (which we can then contribute back to the Docsy theme.) If you agree with the interim solution of making "incompatible" entries bold, then I'll raise an issue asking if anyone wants to prettify this grid (and others to come).

Reply:
I highlighted compatible versions to make the table easier to parse/make it easier find compatible versions.  For checkbox/cross symbols - it may look nicer although in that case we also need to add a description that check means compatible and cross means incompatible.